exampleCourse/questions/element/template/question.html

```diff
@@ -0,0 +1,48 @@
+<p>
+  This page shows has example uses of the <code>pl-template</code>
+  element, including uses of templates within other templates.
+</p>
+
+<hr />
+
+<pl-template file-name="templates/outerTemplate.mustache">
```

Comment:
Let's be consistent with `camelCase` vs `snake_case` in template names.

Reply:
Removed all camelCase in latest push